network-config-analyzer/nca.py

```diff
@@ -25,6 +25,8 @@ def _valid_path(path_location, allow_ghe=False, allowed_platforms=None):
     :rtype: str
     :raises: argparse.ArgumentTypeError when path is invalid
     """
+    if not path_location:
```

Comment:
I added this because the np_list after the query name can be an empty string now.
for example, we can write :
--equiv 
--resource_list <networkpolicy_path>
In this case np_list will be read as an empty string (from const='')
however this check is true only for np_lists so I may add it to the _network_policies_valid_path instead of here (_valid_path), to be more exact